tools/obscuroscan_v3/frontend/src/lib/constants.ts

```diff
@@ -2,12 +2,31 @@ export const socialLinks = {
   github: "https://github.com/ten-protocol",
   discord: "https://discord.gg/2JQ2Z3r",
   twitter: "https://twitter.com/tenprotocol",
+  twitterHandle: "@tenprotocol",
 };
 
 export const pollingInterval = 5000;
 export const maxRetries = 3;
 export const pricePollingInterval = 60 * 1000;
 
+export const getOptions = (query: {
+  page?: string | string[];
+  size?: string | string[];
+}) => {
+  const options = {
+    offset: Number.isNaN(parseInt(query.page as string, 10))
+      ? 1
+      : parseInt(query.page as string, 10),
```

Comment:
And this offset needs to be multiplied by the page size. I know that's a super weird API we should get that fixed but that seems to be how it is atm. So if you're on page 3 and your page size is ten you would send:

offset: 20, size: 10 -> that will give you rows 20-30